src/sysusers/sysusers.c

```diff
@@ -1749,6 +1785,19 @@ static int parse_argv(int argc, char *argv[]) {
                                 return r;
                         break;
 
+                case ARG_CONFIG_NAME:
+                        if (strchr(optarg, '/')) {
+                                log_error("The argument to --name should be a package name: %s", optarg);
```

Comment:
--config-name instead of --name.

Reply:
Indeed. And I added "=" for @poettering ;)